Makefile

```diff
@@ -22,7 +22,11 @@ all: build
 
 volume:
 	echo "Building $(APPNAME) volumes..."
-	sudo docker run -v $(PWD):/opt/$(APPNAME) -v ~/var/$(APPNAME):/var/opt/$(APPNAME) --name $(APPNAME)_volumes busybox true
+	sudo docker run -v $(PWD)/scripts:/var/opt/webhookd/scripts --name $(APPNAME)_volumes busybox true
```

Comment:
Could you keep the first mount point. It's used to run the container in dev mode (aka edit file onto the host)